src/components/ModalStrategy.vue

```diff
@@ -51,6 +53,27 @@ const strategyIsValid = computed(() =>
   strategyValidationErrors.value === true ? true : false
 );
 
+const router = useRouter();
+
+const playgroundLink = computed(() => {
```

Comment:
We have code like this in ModalStrategies, can we reuse that?

Reply:
Weellll..... thought about it too... was a bit lazy. Let's see.... what about a usePlayground composable? :D

Reply:
Ok sounds good